test/test_helper.rb

```diff
@@ -12,10 +12,12 @@ class ActionDispatch::SystemTestCase
 
 VCR.configure do |c|
   c.cassette_library_dir = Rails.root.join('test', 'support', 'vcr')
-  c.default_cassette_options = { :record => :new_episodes }
+  c.default_cassette_options = { record: :new_episodes }
   c.ignore_localhost = true
   c.filter_sensitive_data('<API USERNAME>') { Rails.configuration.x.api_username }
```

Comment:
Since `Rails.configuration.x.api_username` is saved in `username` now, it makes sense to use `username` here as well and change the order of the lines.